spokestack/nlu/tflite.py

```diff
@@ -35,17 +35,21 @@ def __init__(self, model_dir: str) -> None:
         self._intent_meta = {
             intent.pop("name"): intent for intent in self._metadata["intents"]
         }
+        self._slot_meta = {}
+        for intent in self._intent_meta:
+            for slot in self._intent_meta[intent]["slots"]:
+                self._slot_meta[slot.pop("name")] = slot
```

Comment:
In theory, you could have a slot in two different intents that shares a name but has a different type. This would probably be bad for the tagger itself, but I don't think we currently forbid it anywhere. If that showed up in a model, indexing the slots by name here would result in unexpected behavior, since only one type would be parsable.